index.js

```diff
@@ -0,0 +1,15 @@
+var express = require( "express" );
+var app;
+var server;
+var port;
+
+//jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+port = process.env.npm_package_config_port;
+
+//jscs:enable requireCamelCaseOrUpperCaseIdentifiers
+
+app = express();
+app.use ( "/", express.static( "./static" ) );
+server = app.listen( port, function() {
+	console.log( "Listening on http://localhost:" + port );
+} );
```

Comment:
You shouldn't bother implementing an access log, usually that will be provided by whatever software runs in front of this server (e.g. nginx as reverse proxy). Even if this ends up serving requests directly, you can still add logging later.